### DISCUSSION TITLE
Transparency combining render and compose?

### DISCUSSION THREAD

**Discussion by TomJGooding** [Textualize/textual#4293] [Q&A] [answered]
The [Render and compose](https://textual.textualize.io/how-to/render-and-compose/#render-and-compose) How-To says "If you define both, then render will be used as a background".

The composed widget in the example has the "default" background. Is it possible to make this transparent so the background is the render?

**Comment by TomJGooding** [comment]
@willmcgugan Sorry I wouldn't normally tag maintainers in the discussions, but is there any workaround for setting transparency like this?

**Reply by willmcgugan** [reply, marked as answer]
Not currently, no. But you could set the background renderable for widgets where you want the background to show through.

**Reply by TomJGooding** [reply]
That's a shame, thanks anyway Will for confirming.
